core/templates/dev/head/pages/exploration-player-page/learner-experience/input-response-pair.directive.ts

```diff
@@ -87,7 +87,9 @@ angular.module('oppia').directive('inputResponsePair', [
               PlayerPositionService.getDisplayedCardIndex());
             var interaction = displayedCard.getInteraction();
             var shortAnswerHtml = '';
-            if ($scope.data && interaction.id &&
+            if ($scope.data.learnerInput.answerDetails) {
```

Comment:
Just to check -- should this use hasOwnProperty()? Or is answerDetails always set (but possibly null or empty string)?

Reply:
Used hasOwnProperty()